docs/experimental-features/experimental-features-intro.asciidoc

```diff
@@ -1,5 +1,5 @@
 [[sec-experimental-intro]]
-= Experimental features
+= Technical review
```

Comment:
I've seen this as "technical _preview_", per [these guidelines](https://github.com/elastic/docs#using-the-experimental-admonition), https://github.com/elastic/docs/issues/2305, etc.
```suggestion
= Technical preview
```